daemon/api_systems_test.go

```diff
@@ -817,45 +817,48 @@ func (s *systemsSuite) TestSystemsGetSystemDetailsForLabel(c *check.C) {
 	}
 
 	for _, tc := range []struct {
-		disabled, available                bool
-		storageSafety                      asserts.StorageSafety
-		typ                                secboot.EncryptionType
-		unavailableErr, unavailableWarning string
+		disabled, available, passphraseAuthAvailable bool
+		storageSafety                                asserts.StorageSafety
+		typ                                          secboot.EncryptionType
+		unavailableErr, unavailableWarning           string
 
 		expectedSupport                                  client.StorageEncryptionSupport
 		expectedStorageSafety, expectedUnavailableReason string
+		expectedEncryptionFeatures                       []client.StorageEncryptionFeature
 	}{
 		{
-			true, false, asserts.StorageSafetyPreferEncrypted, "", "", "",
-			client.StorageEncryptionSupportDisabled, "", "",
+			true, false, false, asserts.StorageSafetyPreferEncrypted, "", "", "",
+			client.StorageEncryptionSupportDisabled, "", "", nil,
```

Comment:
there aren't that many test cases yet, so maybe use field initializers to avoid mistakes; they aren't awfully longer but are much easier to follow

```suggestion
			disabled: true,
			storageSafety: asserts.StorageSafetyPreferEncrypted,
			expectedSupport: client.StorageEncryptionSupportDisabled,
```